apps/examples/testcase/le_tc/network/tc_net_inet.c

```diff
@@ -102,25 +102,26 @@ static void tc_net_inet_ntoa_p(void)
 static void tc_net_inet_ntop(void)
 {
 	struct in_addr in_addr;
-	char dst[INET_ADDRSTRLEN];
+	char dst4[INET_ADDRSTRLEN];
+	char dst6[INET6_ADDRSTRLEN];
 	const char *ret;
 
 	in_addr.s_addr = 0x17071994;
 
 #ifdef CONFIG_NET_IPv4
-	ret = inet_ntop(AF_INET, &in_addr, dst, INET_ADDRSTRLEN);
+	ret = inet_ntop(AF_INET, &in_addr, dst4, INET_ADDRSTRLEN);
 	TC_ASSERT_NEQ("inet_ntop", ret, NULL);
 
 	/* Failure case: size of destination buffer less than INET_ADDRSTRLEN bytes */
-	ret = inet_ntop(AF_INET, &in_addr, dst, 7);
+	ret = inet_ntop(AF_INET, &in_addr, dst4, 7);
 	TC_ASSERT_EQ("inet_ntop", ret, NULL);
 #endif
 #ifdef CONFIG_NET_IPv6
-	ret = inet_ntop(AF_INET6, &in_addr, dst, INET_ADDRSTRLEN);
+	ret = inet_ntop(AF_INET6, &in_addr, dst6, INET6_ADDRSTRLEN);
```

Comment:
also, Is not this the second argument "in6_addr"?

Reply:
You're right. I added "in6_addr" for IPv6.